### PR TITLE
POA added, deploys endpoint, registry, secret registry contracts.

### PR DESCRIPTION
**Pull request by anmolshl**
Very simple changes. POA added, deploys endpoint, registry, secret registry contracts.

### CODE REVIEW

**Comment by anmolshl** [comment]
@palango check this pull request please! Thank you

File: raiden_contracts/deploy/__main__.py [hunk 1]
```diff
@@ -98,21 +102,32 @@ def main(**kwargs):
     if gas_price == 0:
         transaction['gasPrice'] = gas_price
 
+    from pprint import pprint
```
Comment (palango) [inline raiden_contracts/deploy/__main__.py]: this is not necessary

File: raiden_contracts/deploy/__main__.py [hunk 2]
```diff
@@ -98,21 +102,32 @@ def main(**kwargs):
...
-        if not token_address:
-            token_address = deploy_contract(
-                web3,
-                contracts_compiled_data,
-                'CustomToken',
-                transaction,
-                txn_wait,
-                [supply, token_decimals, token_name, token_symbol],
-            )
+        # if not token_address:
```
Comment (palango) [inline raiden_contracts/deploy/__main__.py]: should not be commented out.

File: raiden_contracts/deploy/__main__.py [hunk 3]
```diff
@@ -122,6 +137,8 @@ def main(**kwargs):
             txn_wait,
         )
 
+        time.sleep(2)
```
Comment (palango) [inline raiden_contracts/deploy/__main__.py]: not needed

File: raiden_contracts/deploy/__main__.py [hunk 4]
```diff
@@ -142,18 +159,18 @@ def main(**kwargs):
...
-        )
-        receipt = check_succesful_tx(web3, txhash, txn_wait)
-
-        print(
-            'TokenNetwork address: {0}. Gas used: {1}'.format(
-                token_network_registry.functions.token_to_token_networks(token_address).call(),
-                receipt['gasUsed'],
-            ),
-        )
+        #
```
Comment (palango) [inline raiden_contracts/deploy/__main__.py]: Should not be commented out.

File: raiden_contracts/deploy/__main__.py [hunk 5]
```diff
@@ -98,21 +102,32 @@ def main(**kwargs):
...
-            )
+        # if not token_address:
+        #     token_address = deploy_contract(
+        #         web3,
+        #         contracts_compiled_data,
+        #         'CustomToken',
+        #         transaction,
+        #         txn_wait,
+        #         [supply, token_decimals, token_name, token_symbol],
+        #     )
```
Comment (loredanacirstea) [inline raiden_contracts/deploy/__main__.py]: Why is this commented out?

File: raiden_contracts/deploy/__main__.py (hunk 3 above)
Comment (loredanacirstea) [inline raiden_contracts/deploy/__main__.py]: Why is this needed? We use `check_succesful_tx`, that waits for the transaction to be mined.

File: raiden_contracts/deploy/__main__.py [hunk 6]
```diff
@@ -142,18 +159,18 @@ def main(**kwargs):
...
+        #     token_address,
+        # )
+        # receipt = check_succesful_tx(web3, txhash, txn_wait)
+        #
+        # print(
+        #     'TokenNetwork address: {0}. Gas used: {1}'.format(
+        #         token_network_registry.functions.token_to_token_networks(token_address).call(),
+        #         receipt['gasUsed'],
+        #     ),
+        # )
```
Comment (loredanacirstea) [inline raiden_contracts/deploy/__main__.py]: Why is this commented out?

**Comment by anmolshl** [comment]
@palango @loredanacirstea sorry about that. New commits are up, should be fixed now!

**Comment by palango** [comment]
Looks better now, please just squash together the commits into a single one.

**Comment by anmolshl** [comment]
@palango done!

**Comment by palango** [comment]
@anmolshl Looks good. Please give the commit a better desciption, then we can merge this.

**Comment by anmolshl** [comment]
@palango commit description changed.
